examples/src/main/java/jaxrs/examples/client/BasicExamples.java

```diff
@@ -20,6 +21,9 @@
 import jakarta.ws.rs.WebApplicationException;
 import jakarta.ws.rs.client.Client;
 import jakarta.ws.rs.client.ClientBuilder;
+import static jakarta.ws.rs.client.Entity.form;
```

Comment:
I consider *mixing* `import` and `static import` lines an anti-pattern, as it is easy to miss something. Also, IDEs often sort this automatically. So if this is the default of Jakarta, I'm OK with it. But if this happened by incident or just because it is your personal choice, I would ask to not do that. Same with having `javax` *before* `java`.

Reply:
This was unintentional, default settings in IntelliJ are weird. Will fix.